src/components/Header.js

```diff
@@ -1,12 +1,40 @@
 import React, { Component } from "react";
+import { Nav, NavItem, NavLink, Jumbotron, Container  } from 'reactstrap';
 
 class Header extends Component {
 
     render () {
         return (
             <>
-            <h1>Header </h1>
-                
+            <div>
+                <Nav className="navbar">
+                    <NavItem>
+                    <NavLink href="#">Home</NavLink>
```

Comment:
```suggestion
                        <NavLink href="#">Home</NavLink>
```
NavLinks need to be nested with indentation